### ISSUE TITLE
jsdoc-vuejs not compatible with JavaScript modules?

### ISSUE THREAD

**Issue by pkuwwt** [Kocal/jsdoc-vuejs#136]
JSDoc-vuejs is great. But when I use it, the comments for function in a module will disappear.

For example, I create a vue project:
```
vue init jsdoc-test
cd jsdoc-test
npm install -S jsdoc jsdoc-vuejs
```

And then I create the configuration file `jsdoc.config.json`:

```json
{
  "plugins": [
    "node_modules/jsdoc-vuejs"
  ],
  "source": {
    "includePattern": "\\.(vue|js)$"
  }
}
```

There are two modules: `test-es.js` and `test-common.js`:

```javascript test-es.js
/**
 * @module /test-es
 */

/**
 * print hello
 */
function hello () {
	console.log('hello');
}
export default {
	hello
};

```

```javascript test-common.js
/**
 * @module /test-common
 */

module.exports = {
	/**
	 * @param {string} i - parameter to be printed
	 */
	test: function(i) {
		console.log(i);
	}
};
```

Then I test them with and without `jsdoc.config.json`:

```
jsdoc -c jsdoc.config.json test-es.js test-common.js
jsdoc test-es.js test-common.js
```

The results are different: comments for the methods disappear if I use `jsdoc.config.json`.

Do I miss something or is there a real issue?

**Comment by Kocal** [comment]
Hum, I'm pretty sure it's because of https://github.com/Kocal/jsdoc-vuejs/pull/128...
Which version do you use?

**Comment by pkuwwt** [comment]
> Hum, I'm pretty sure it's because of #128...
> Which version do you use?

jsdoc 3.5.5
jsdoc-vuejs 2.3.0

**Comment by Kocal** [comment]
I mean #128 probably introduce the problem.
Can you try with `jsdoc-vuejs` please?

**Comment by pkuwwt** [comment]
> I mean #128 probably introduce the problem.
> Can you try with `jsdoc-vuejs` 2.2.3?

Yes, version 2.2.3 is fine

**Comment by Kocal** [comment]
Okay I see the problem.
I'm working on a patch.

**Comment by Kocal** [comment]
@pkuwwt PR #137 should fix this issue. Would you mind to try it?

**Comment by pkuwwt** [comment]
> @pkuwwt PR #137 should fix this issue. Would you mind to try it?

It works!.
Thanks.

**Comment by Kocal** [comment]
Released in v2.3.1
